src/Storage/Storage/Blob/Cmdlet/GetAzureStorageBlob.cs

```diff
@@ -329,6 +329,7 @@ internal async Task ListBlobsByPrefix(long taskId, IStorageBlobManagement localC
                     {
                         if (blobFilter == null || blobFilter(item.Name))
                         {
+                            ClientOptions.TrimBlobNameSlashes = false;
```

Comment:
Not sure if this really works. Have you tested the TrimBlobNameSlashes = false here works?

Reply:
It works for the cases of listing blobs below - 
![image](https://user-images.githubusercontent.com/100746763/230258909-c57c0ce0-4187-48e6-91a9-e6563375af93.png)

Reply:
One case that is still not working is getting a single blob with a leading slash in the name, say, Get-AzStorageBlob -Container $containerName -Blob //aaa/test1 
As the fix only applies for listing blobs, this command will still result in blob not found

Reply:
For single blob, there's no regression, might be OK.
However, if you can also fix single blob one, it should be better.
(Or if we consider this as breaking, might can add a breaking notice in 4/25 release, then add the single blob implemetation on May release.)

Reply:
Yes I'm thinking that for single blob, this might be a breaking change for the following scenario: 
If the customer has both "//aa/test1" and "aa/test1", then when getting a single blob with input "//aa/test1", currently the cmdlet will return "aa/test1" instead of "//aa/test1".

Will add the breaking notice in another PR